doc/how-to-contribute.md

```diff
@@ -0,0 +1,117 @@
+# How to contribute
+
+The way to make contributions is through making pull requests(PR for short). After your PR is merged, the changes you make can be applied by other users.
+
+<div align=center><href="https://sm.ms/image/qt1MWOfYbQzKGJC" target="_blank"><img src="https://s2.loli.net/2022/08/16/qt1MWOfYbQzKGJC.png" width="60%"></div>
+
+Firstly, fork in DP-GEN repository. Then you can clone the repository, build a new branch, make changes and then make a pull request.
+
+---
+
+## How to contribute to DP-GEN
+
+DP-GEN adopts the same convention as other softwares in DeepModeling Community. 
+
+You can first refer to DeePMD-kit's
+[Contributing guide](https://github.com/deepmodeling/deepmd-kit/edit/devel/CONTRIBUTING.md)
+and [Developer guide](https://github.com/deepmodeling/deepmd-kit/edit/devel/doc/development/index.md).
+
+Welcome to the repository of DP-GEN https://github.com/deepmodeling/dpgen
+
+  If you have no idea how to fix your problem or where to find the relative code, see OVERVIEW/Overview of the structure of the DP-GEN repository on this website.
+  
+### Use command line
+You can use git with the command line, or open the repository on Github Desktop. Here is a video as a demo of making changes to DP-GEN and publishing it with command line.
+
+https://www.youtube.com/watch?v=DPApc1_eNS8
```

Comment:
You can use my script [bilitube](https://github.com/njzjz/bilitube) to automatically switch between bilibili and youtube.